Documentation/btrfs-scrub.rst

```diff
@@ -96,6 +96,13 @@ start [-BdrRf] <path>|<device>
                 can avoid writes from scrub.
         -R
                 raw print mode, print full data instead of summary
+	-t <throughput_limit>
+		set the scrub throughput limit for each device, acts the same as
+		``btrfs scrub limit -a -l <throughput_limit>``.
+
+		The value is bytes per second, and accept the usual KMGT prefixes.
+		After the scrub is finished, the throughput limit will be reset to
+		zero (aka, no limit).
```

Comment:
What if there already is some limit set, should it be reset to the original value instead?

Reply:
That's a good point.

I'll save and reset to the old values instead.